app/classes/amity.py

```diff
@@ -425,6 +440,11 @@ def load_state(self, db_name='default'):
 		Session.configure(bind=engine)
 		session = Session()
 
+		self.people = []
```

Comment:
Does this overwrite all the data that i had in my app and have not saved to the database?